thresher/models.py

```diff
@@ -78,13 +87,27 @@ def __unicode__(self):
         return "Topic %s in Analysis Type %s" % (self.name, self.analysis_type.name) 
 
 # The question in a given topic
-class Question(models.Model):
+class QuestionUnderTopic(models.Model):
     # an id within the given topic
-    question_id = models.IntegerField()
+    question_id = models.OneToOneField(QuestionContent)
 
     # The topic this question belongs to
-    topic = models.ForeignKey(Topic, related_name="questions")
+    topic_id = models.ForeignKey(Topic, related_name="related_questions")
+
+    # The order of the question comparing to other questions under the same topic
+    order = models.IntegerField()
     
+    class Meta:
```

Comment:
This class Meta is saying that the question_id and topic_id must be unique when considered together. 
I don't know if my interpretation is correct according to this documentation (https://docs.djangoproject.com/en/1.9/ref/models/options/):
![image](https://cloud.githubusercontent.com/assets/5152856/13408258/bd60049a-dee1-11e5-8381-6855804ce839.png)